tools/mininet/stratum.py

```diff
@@ -276,6 +276,8 @@ def config(self, **params):
             cmd = "/sbin/ethtool --offload %s %s off" \
                   % (self.defaultIntf(), off)
             self.cmd(cmd)
+        cmd = "sysctl net.ipv6.conf.%s.disable_ipv6=1" % (self.defaultIntf())
+        self.cmd(cmd)
```

Comment:
I'm not sure how the `hosts` export for mininet works, but shouldn't this command be it's own class? Like `NoIpv6Host`?

Reply:
Good point will get that fixed up 👍